### PR TITLE
Implement Input

### PR DESCRIPTION
**Pull request by hirwaaldo1**
**Feature**
1. different variant = "normal" | "fill" | "outlined" ✅
2. different  size = "sm" | "md" | "lg"✅
3. different rounded =  "sm" | "md" | "lg" | "full" | "none" ✅
4. disabled = boolean✅
5. placeholder = string✅
6. invalid = boolean✅
7. label = string✅
8. onChange = Functiom✅
9. type = string✅

### CODE REVIEW

File: site/app/page.tsx [hunk 1]
```diff
@@ -10,19 +10,67 @@ export default function Page() {
   const [autoCompleteValue, setautoCompleteValue] = useState<string>();
   const [multiValues, setmultiValues] = useState<any>([]);
```
Comment (edson-pro) [inline site/app/page.tsx:11]: if array of use ts typing with any[]

File: site/app/page.tsx [hunk 2]
```diff
@@ -10,19 +10,67 @@ export default function Page() {
...
   const [checked, setchecked] = useState(false);
-
+  const [variantvalue, setVariantValue] = useState<any>("normal");
```
Comment (edson-pro) [inline site/app/page.tsx]: these can be typed as string instead of any

File: site/components/ui/Input.tsx [hunk 3]
```diff
@@ -1,72 +1,96 @@
-import React, { useState } from "react";
```
Comment (edson-pro) [inline site/components/ui/Input.tsx:1]: You should add a error message below the input which can be red in color

File: site/tailwind.config.js [hunk 4]
```diff
@@ -8,7 +8,12 @@ module.exports = {
   theme: {
     extend: {
       colors: {
-        primary: "#6271EB",
+        primary: {
```
Comment (edson-pro) [inline site/tailwind.config.js:11]: what is the use of those primary object??

**Review by edson-pro — CHANGES_REQUESTED** [review]
Good one.
fix those few comments!!

File: site/tailwind.config.js (hunk 4 above)
Comment (hirwaaldo1) [reply site/tailwind.config.js:11]: I use object because it has more than  one color

Comment (edson-pro) [reply site/tailwind.config.js:11]: I think if there are different primary you can use more descriptive names like primaryLight or primaryDark things like that

**Review by edson-pro — APPROVED** [review]

**Pull request merged**